gammapy/stats/feldman_cousins.py

```diff
@@ -0,0 +1,444 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Provide a Feldman Cousins algorithm."""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+import numpy as np
+from astropy.extern.six.moves import range
+from astropy.extern.six import iteritems
+
+import logging
+log = logging.getLogger(__name__)
+
+__all__ = ['fc_find_acceptance_region_gauss',
```

Comment:
I replaced all the "acceptance_region" with "acceptance_interval" (which is also what is used in the FC paper.

I already explained this in an earlier comment: fc_find_acceptance_interval_poisson does not give you an confidence interval, but an acceptance_interval. So I think the function names are fine.